tests/system/src/test/java/test/robot/helloworld/CustomSecurityManagerTest.java

```diff
@@ -194,18 +194,18 @@ private void doTestFullScreenCommon(SecurityManager sm,
             } else {
                 assertFalse(propertyState);
             }
+            final int offset = 10;
             for (int row = 0; row < 2; row++) {
-                int h = screenHeight.get();
                 int y;
                 if (row == 0) {
                     // avoid the top area as it might contain OS-specific UI (Macs with a notch)
-                    y = h / 3;
+                    y = (int)screenBounds.getMinY() + offset;
                 } else {
-                    y = h - 2;
+                    y = (int)screenBounds.getMaxY() - offset - 1;
                 }
```

Comment:
The comment is now incorrect. Maybe you can delete the comment and replace this block with similar logic as used for `x` below?